src/main/java/pl/edu/mimuw/BankClient.java

```diff
@@ -1,35 +1,73 @@
 package pl.edu.mimuw;
 
+import java.util.ArrayList;
 import java.util.List;
 
-public class BankClient {
+public class BankClient
+{
+	private final String name;
+	private final String surname;
+	private final int age;
+	private final String accountNumber;
+	private double money;
+	private final String currency;
+	private ArrayList <BankAction> actionHistory;
```

Comment:
Prawie idealnie sformatowane gdyby nie ta spacja xd Zdecydowanie polecam automatycznie co pare sekund to robić 😉